### ISSUE TITLE
Color contrast of disabled form controls

### ISSUE THREAD

**Issue by isner** [dequelabs/axe-core#41]
A color contrast violation should not be identified for labels of inputs which are currently disabled, per the "Incidental" exception found at http://www.w3.org/TR/WCAG20/#visual-audio-contrast-contrast.

For example, in the example below, the label containing "Jim" should not fail color contrast.
```html
<style>
  label.disabled {
    color: #ccc;
  }
</style>

<input id="bob" type="radio" name="name" />
<label for="bob">Bob</label>

<input id="jim" type="radio" name="name" disabled="disabled" />
<label for="jim" class="disabled">Jim</label>
```
